Documentation~/Blender.md

```diff
@@ -69,3 +70,36 @@ This option will only appear if we install [ProBuilder](https://docs.unity3d.com
 > If **Bake Modifiers** option is enabled in the plugin, 
 > a mesh that is sent from Unity back into Blender will have modifiers applied beforehand and have them removed after.
 
+## Material sync mode
+
+The MeshSyncServer in Unity always creates a material with a default shader for the active render pipeline.
+There are multiple options to export materials from blender:
+
+### None
+
+Only the material names are exported to Unity and default materials created. Users can override the materials in the mapping on the MeshSyncServer to use their own materials.
+
+### Basic
+
+The materials are synced to Unity in a limited way.
+MeshSync looks for an active material output node and exports the BSDF connected to that.
```

Comment:
The explanation of the basic option is not clear enough in my opinion because there are a lot of ifs.

Some ideas that come to my mind.
1. Clearly separate what happens if a BSDF is found or not 
2. Put and describe what happens to each socket in a table 
3. Clearly put what is supported, what is partially supported, and what is not supported. Something like [this](https://github.com/Unity-Technologies/MeshSyncDCCPlugins/blob/fe9abd792916acaa87049ada5e3114f8995a4d4c/Documentation~/index.md?plain=1#L9)

Reply:
Done